### ISSUE TITLE
Memory Leak Issue

### ISSUE THREAD

**Issue by azampagl** [jsplumb/community-edition#126]
**_TL;DR:**_ jsPlumb objects remain in memory even after you execute commands such as **deleteEveryEndpoint()**.

Executing "cleanup" commands such as **deleteEveryEndpoint()** successfully removes the connections on the rendered plumb diagram, but do not free the JavaScript objects from memory.

A demo of the memory leak can be found here: http://jsfiddle.net/F5pLC/5/

After running the demo, use Chrome's Heap Snapshot tool (found in Chrome's Dev Tools).  You will notice that the jsPlumb.Endpoint and jsPlumb.Connection objects remain even though they were told to be "deleted".

An image of the Chrome Heap Snapshot tool can be found here: http://i.imgur.com/eqLf6zD.png

Notice that object counts for _jsPlumb.Endpoint_ and _jsPlumb.Connection_ are > 0.  They should not exist at all.

This bug eventually causes Internet Explorer to lock up and Firefox to get a Stack Overflow.
